File: platforms/python/examples/scene/camera/view_proj.py
from fragmentcolor import Camera

camera = Camera.perspective(1.047, 16.0 / 9.0, 0.1, 100.0).look_at([0.0, 0.0, 5.0], [0.0, 0.0, 0.0], [0.0, 1.0, 0.0])

view_proj = camera.view_proj()

File: platforms/kotlin/examples/scene/camera/view_proj.kt
import org.fragmentcolor.*

val camera = Camera.perspective(1.047f, 16.0f / 9.0f, 0.1f, 100.0f).lookAt(listOf(0.0f, 0.0f, 5.0f), listOf(0.0f, 0.0f, 0.0f), listOf(0.0f, 1.0f, 0.0f))

val view_proj = camera.viewProj()